app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java

```diff
@@ -166,6 +173,13 @@ public void run() {
                 });
             }
         }, Message.IMAGE);
+
+        mConnectivityStatusReceiver = new ConnectivityStatusReceiver();
+        mConnectivityStatusReceiver.setListener(this);
+
+        IntentFilter intentFilter = new IntentFilter(ConnectivityStatusReceiver.INTENT_NAME);
```

Comment:
Double registration of receiver.
registration should be done on onResume alone !resolve